### ISSUE TITLE
Vimode does not work on the split screen

### ISSUE THREAD

**Issue by salil19** [geany/geany-plugins#1273]
Hi,
When I open Geany and I have single screen, the vimode plugin works fine. When I split the screen, either horizontal or vertical, the vimode plugin does work on the newly opened screen. It continues to open in the original screen.
Can you please help me resolve the issue. Here are the details:
16:11:04.448133: GLib-GIO DEBUG	: _g_io_module_get_default: Found default implementation gvfs (GDaemonVfs) for ‘gio-vfs’
16:11:04.449198: Geany INFO		: Geany 1.38, en_IN
16:11:04.449203: Geany INFO		: GTK 3.24.33, GLib 2.72.4
16:11:04.449216: Geany INFO		: OS: Linux Mint 21.2 (victoria)
16:11:04.449219: Geany INFO		: System data dir: /usr/local/share/geany
16:11:04.449221: Geany INFO		: User config dir: /home/salil/.config/geany
16:11:04.547968: Geany INFO		: Loaded GTK+ CSS theme '/usr/local/share/geany/geany.css'
16:11:04.548445: Geany INFO		: Loaded GTK+ CSS theme '/usr/local/share/geany/geany-3.20.css'
16:11:04.551359: Geany INFO		: System plugin path: /usr/local/lib/geany
16:11:04.555505: Geany INFO		: Added filetype Arduino (64).
16:11:04.555510: Geany INFO		: Added filetype Swift (65).
16:11:04.555513: Geany INFO		: Added filetype Graphviz (66).
16:11:04.555516: Geany INFO		: Added filetype Kotlin (67).
16:11:04.555518: Geany INFO		: Added filetype Groovy (68).
16:11:04.555521: Geany INFO		: Added filetype Genie (69).
16:11:04.555524: Geany INFO		: Added filetype Cython (70).
16:11:04.555526: Geany INFO		: Added filetype Meson (71).
16:11:04.555529: Geany INFO		: Added filetype CUDA (72).
16:11:04.555531: Geany INFO		: Added filetype Clojure (73).
16:11:04.555534: Geany INFO		: Added filetype JSON (74).
16:11:04.555537: Geany INFO		: Added filetype Scala (75).
16:11:04.555539: Geany INFO		: Added filetype TypeScript (76).
16:11:04.555542: Geany INFO		: Added filetype Nim (77).
16:11:04.562060: Geany INFO		: Loaded libvte from libvte-2.91.so.0
16:11:04.573804: Geany INFO		: unknown : None (UTF-8)

**Comment by elextr** [comment]
Split screen is a plugin and vimode is a plugin.  

Plugins do not know about the existence of other plugins so its unsurprising that vimode does not know about the second screen.

**Comment by techee** [comment]
Yes, I'm afraid this is currently "unfixable" unless we add some split screen support directly to Geany.
